Cython/Compiler/ExprNodes.py

```diff
@@ -10156,6 +10156,10 @@ class CodeObjectNode(ExprNode):
     is_temp = False
     result_code = None
 
+    tuple_index_re = re.compile(
+        rf"{Naming.pyrex_prefix}tuple\[(\d+)\]"
```

Comment:
There's definitely better ways of doing that. I just don't really want to spend too much time on it until we've decided if this is worthwhile or should be abandoned as a marginal gain.